### PR TITLE
Field name changes fixed

### LINKED ISSUE #11: Field not found error

### LINKED ISSUE #12: Risk values changed in Aquifer_RechargeDischargeAreas
field values used in risk assessment valuation have changed.

### PR DESCRIPTION
**Pull request by keatonwalker**
Field name and risk value changes fixed. This closes #11 and closes #12 .

### CODE REVIEW

File: src/tank_risk.py [hunk 1]
```diff
@@ -453,24 +464,24 @@ def start(self, tankPnts, mapDocument):
...
         resultRows = TankResult.getOutputRows(self.riskFeatureNameOrder)
         self.createOutputTable(resultRows)
-        
+
 
 if __name__ == '__main__':
-    
-    version = "0.7.1"
+
+    version = "1.0.1"
```
Comment (steveoh) [inline src/tank_risk.py]: when you bump minor, patch resets. When you bump major, minor and patch reset. This should be 1.0.0 not that it matters much.

File: src/tank_risk.py [hunk 2]
```diff
@@ -410,41 +410,52 @@ class TankRisk(object):
...
+        # Check for missing fields in attribute layers.
+        for riskFeature in riskFeatures:
+            featureName = self.parseName(riskFeature)
+            if featureName in TankResult.attributesForFeature and TankResult.attributesForFeature[featureName].type == TankResult.ATTRIBUTE:
+                fieldNames = [f.name for f in arcpy.ListFields(riskFeature)]
+                calcFields = TankResult.attributesForFeature[featureName].calcFields
+                for cF in calcFields:
+                    if cF not in fieldNames:
+                        arcpy.AddError("Field {} not present in {}".format(cF, featureName))
+                        return
```
Comment (steveoh) [inline src/tank_risk.py]: Do you want to return here? What about multiple bad fields?

**Comment by steveoh** [comment]
LGTM

**Comment by steveoh** [comment]
Let me know your thoughts on this stuff @keatonwalker so we can get this merged. Its been about a week...

File: src/tank_risk.py (hunk 2 above)
Comment (keatonwalker) [inline src/tank_risk.py]: I changed this up a little so it does alert when multiple fields are missing.

**Comment by steveoh** [comment]
Will you git tag this as v1.0.0 and git push --tags.

**Pull request merged**